darts/tests/test_torch_forecasting_model.py

```diff
@@ -0,0 +1,142 @@
+            model_name = 'test_model'
+            model1 = RNNModel(10, 10, work_dir=self.temp_work_dir, model_name=model_name)
+            # no exception is raised
+
+            times = pd.date_range('20130101', '20130410')
+            pd_series = pd.Series(range(100), index=times)
+            series = TimeSeries.from_series(pd_series)
+            model1.fit(series, epochs=1)
+
+            model2 = RNNModel(10, 10, work_dir=self.temp_work_dir, model_name=model_name, force=True)
+            model2.reset_model()
+            model2.fit(series, epochs=1)
+            # no exception is raised
+
+        # n_epochs=20, fit|epochs=None, total_epochs=0 - train for 20 epochs
+        def test_train_from_0_to_20_no_fit_epochs(self):
+            model1 = RNNModel(10, 10, n_epochs=20, work_dir=self.temp_work_dir)
+
+            times = pd.date_range('20130101', '20130410')
+            pd_series = pd.Series(range(100), index=times)
+            series = TimeSeries.from_series(pd_series)
+            model1.fit(series)
+
+            self.assertEqual(model1.total_epochs, 20)
+
+        # n_epochs = 20, fit|epochs=None, total_epochs=20 - train for another 20 epochs
+        def test_train_from_20_to_40_no_fit_epochs(self):
+            model1 = RNNModel(10, 10, n_epochs=20, work_dir=self.temp_work_dir)
+
+            times = pd.date_range('20130101', '20130410')
+            pd_series = pd.Series(range(100), index=times)
+            series = TimeSeries.from_series(pd_series)
+            model1.fit(series)
+            self.assertEqual(model1.total_epochs, 20)
+
+            model1.fit(series)
+            self.assertEqual(model1.total_epochs, 40)
+
+        # n_epochs = 20, fit|epochs=None, total_epochs=10 - train for another 20 epochs
+        def test_train_from_10_to_20_no_fit_epochs(self):
+            model1 = RNNModel(10, 10, n_epochs=20, work_dir=self.temp_work_dir)
+
+            times = pd.date_range('20130101', '20130410')
+            pd_series = pd.Series(range(100), index=times)
+            series = TimeSeries.from_series(pd_series)
+            model1.fit(series, epochs=10)  # cheating here a little bit
+            self.assertEqual(model1.total_epochs, 10)
+
+            model1.fit(series)
+            self.assertEqual(model1.total_epochs, 30)
+
+        # n_epochs = 20, fit|epochs=15, total_epochs=0 - train for 15 epochs
+        def test_train_from_0_to_20_fit_15_epochs(self):
+            model1 = RNNModel(10, 10, n_epochs=20, work_dir=self.temp_work_dir)
+
+            times = pd.date_range('20130101', '20130410')
+            pd_series = pd.Series(range(100), index=times)
+            series = TimeSeries.from_series(pd_series)
+            model1.fit(series, epochs=15)
+            self.assertEqual(model1.total_epochs, 15)
+
+        # n_epochs = 20, fit|epochs=15, total_epochs=10 - train for 15 epochs
+        def test_train_from_0_to_20_fit_15_epochs(self):
```

Comment:
I like the tests, but I don't fully grasp the logic behind the names here

Reply:
in the case above: 0 is the current number of epochs, number after `_to_` is the `n_epochs` parameter and then either `epochs` value for `fit`. Do you have other suggestion?